src/main/java/bio/terra/service/tabulardata/azure/AzureStorageTablePdao.java

```diff
@@ -129,7 +131,7 @@ public void storeLoadHistory(
   private static TableEntity bulkFileLoadModelToStorageTableEntity(
       StorageTableLoadHistoryEntity entity, String loadTag, Instant loadTime) {
     var model = entity.model;
-    return new TableEntity(entity.partitionKey, model.getFileId())
+    return new TableEntity(entity.partitionKey, UUID.randomUUID().toString())
```

Comment:
TODO: determine what we want to set this to since file id is sometimes null. Currently discussing in slack.

Reply:
@tlangs  - Did you get a chance to look at this change? Description in old slack message: https://broadinstitute.slack.com/archives/C01VBGH431S/p1629760679215600

Reply:
Hmm, that's annoying, since StorageTables use the row id for ordering, and we want the ordering to be consistent between GCP and Azure. The hash of the target path might be a better idea.

Reply:
Ticket created here: https://broadworkbench.atlassian.net/browse/DR-2074